starcheck/pcad_att_check.py

```diff
@@ -0,0 +1,44 @@
+import hopper
+
+def make_pcad_attitude_check_report(backstop_file, or_list_file=None,
+                                    ofls_characteristics_file=None, out=None,
+                                    ):
+    """
+    Make a report for checking PCAD attitudes
+
+    """
+
+    # Run the commands and populate attributes in `sc`, the spacecraft state.
+    # In particular sc.checks is a dict of checks by obsid.
+    # Any state value (e.g. obsid or q_att) has a corresponding plural that
+    # gives the history of updates as a dict with a `value` and `date` key.
+    # This does not define the initial state from continuity at this time.
+    sc = hopper.run_cmds(backstop_file, or_list_file, ofls_characteristics_file,
+                         initial_state=None)
```

Comment:
I seem to recall that the attitude history is probably parseable as a
table.  Should we get the sim history for the initial state too?
On Nov 11, 2015 4:24 PM, "Tom Aldcroft" <notifications@github.com> wrote:

> In starcheck/pcad_att_check.py
> <https://github.com/sot/starcheck/pull/122#discussion_r44588297>:
>
> > +
> > +def make_pcad_attitude_check_report(backstop_file, or_list_file=None,
> > +                                    ofls_characteristics_file=None, out=None,
> > +                                    ):
> > +    """
> > +    Make a report for checking PCAD attitudes
> > +
> > +    """
> > +
> > +    # Run the commands and populate attributes in `sc`, the spacecraft state.
> > +    # In particular sc.checks is a dict of checks by obsid.
> > +    # Any state value (e.g. obsid or q_att) has a corresponding plural that
> > +    # gives the history of updates as a dict with a `value` and `date` key.
> > +    # This does not define the initial state from continuity at this time.
> > +    sc = hopper.run_cmds(backstop_file, or_list_file, ofls_characteristics_file,
> > +                         initial_state=None)
>
> Seems like a job for parse_cm.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sot/starcheck/pull/122/files#r44588297>.
>